CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+Pull requests are welcome from anyone, please feel free to contribute by following the below guidelines.
+
+Fork, then clone the repo:
+
+    git clone git@github.com:your-username/publishers.git
+
+Write your code and tests, then make sure the tests pass:
+
+    bin/rake test
+
+Push to your fork and [submit a pull request][pr].
+
+[pr]: https://github.com/brave-intl/publishers/compare/
+
+We like to make sure that all contributions follow a few guidelines, which will increase the likelihood of a merge:
+
+- Follow the [code review guildelines][codereview]
+- Write tests to cover critical functionality
+- Write a [good commit message][commit]
+- Leave code better than you found it, whenever possible:
+  - eliminate [callbacks][callbacks], use a service object instead
+  - use [dependency injection][di] to encourage modularity, [an example][di-ex]
+  - follow Sandi Metz' [rules][sm]
+  - prefer [composition over inheritance][composition]
+
+[composition]: https://betterprogramming.pub/prefer-composition-over-inheritance-1602d5149ea1
```

Comment:
IMO the article is dogmatic, other's like such as https://thoughtbot.com/blog/reusable-oo-composition-vs-inheritance suggest that the developer should understand the trade-off between composition vs inheritance.

Reply:
I agree, that's why I put prefer.